src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManager.php

```diff
@@ -236,7 +236,7 @@ public function getTree($locale, $offset, $limit, $search, $depth = 0, $sortBy =
 
         unset($filter['limit']);
         unset($filter['offset']);
-        $this->count = $this->collectionRepository->count($depth, $filter);
```

Comment:
That's required because of a BC break in Doctrine, right?

Reply:
yes